StatPlugin.cs

```diff
@@ -1,40 +1,52 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using ServiceStack.Redis;
 
 namespace DNWS
 {
   class StatPlugin : IPlugin
   {
     protected static Dictionary<String, int> statDictionary = null;
+    protected static RedisManagerPool manager = null;
     public StatPlugin()
     {
-      if (statDictionary == null)
+      if (manager == null)
       {
-        statDictionary = new Dictionary<String, int>();
+        manager = new RedisManagerPool("localhost:6379");
```

Comment:
You should not assume that redis will run in localhost, because it's a swarm, web and redis might run in different host.